Prepare unixfs release v0.1.0 (#313)
rs-ipfs/rust-ipfs: unixfs/CHANGELOG.md

# 0.1.0

> Minor version bump for so many new features and broken API.

Reviewer: ```suggestion
Minor version bump due to many new features and broken API.
```
